app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "fr.gaulupeau.apps.InThePoche"
-        minSdkVersion 11
+        minSdkVersion 19
```

Comment:
My idea was to use getExternalFilesDirs() if we're on SDK >= 19 and default to internal emulated external storage in the other case.
According to Google Play, we have something like 5% of users below KitKat (and the F-Droid numbers are certainly higher), so I'll prefer to have an special case just for this.